apps/studio/src/components/connection/AddDatabaseForm.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <form class="save-connection expand" @submit.prevent="save">
+    <h3 class="dialog-c-title">Add database</h3>
+
+    <div v-if="error" class="alert alert-danger">
+      {{error}}
+    </div>
+
+    <div class="form-group">
+      <input class="form-control" v-model="databaseName" @keydown.enter.prevent.stop="save" type="text" placeholder="Database Name">
+    </div>
+
+    <div class="form-group" v-if="charSets.length > 0">
+      <select v-model="selectedCharset">
+        <option v-for="charset in charSets" v-bind:key="charset" :selected="charset === defaultCharSet" :value="charset">{{charset}}</option>
```

Comment:
Just as an FYI `v-bind:key=` is the same as `:key=`

Reply:
Should this attribute be `defaultCharSet` or `selectedCharSet`? I think it should be `selected`

Reply:
relic of the previous PR for the first one, can def change it. Good eye as I missed it.

Reply:
Also switched it over to `selectedCharset` because that gets set on mounted anyway to the default one. Not sure why `defaultCharset` was even there.